SCARA_T7:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  

  # Position Controllers ---------------------------------------
  joint1_position_controller:
    type: effort_controllers/JointPositionController
    joint: Joint_1
    pid: {p: 1000.0, i: 0.00, d: 20.0}

  joint2_position_controller:
    type: effort_controllers/JointPositionController
    joint: Joint_2
    pid: {p: 500.0, i: 0.00, d: 10.0}
    
  # joint3_position_controller:
  #   type: effort_controllers/JointPositionController
  #   joint: Joint_3
  #   pid: {p: 6000.0, i: 0.00, d: 40.0}

  # Velocity Controllers ---------------------------------------
  joint1_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: Joint_1
    pid: {p: 40.0, i: 0.00, d: 0}

  joint2_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: Joint_2
    pid: {p: 6.0, i: 0.00, d: 0}
    
  # joint3_velocity_controller:
  #   type: effort_controllers/JointVelocityController
  #   joint: Joint_3
  #   pid: {p: 100.0, i: 0.00, d: 10.0}